improver/utilities/copy_attributes.py

```diff
@@ -26,25 +26,24 @@ def __init__(self, attributes: List):
         self.attributes = attributes
 
     def process(
-        self, *cubes: Union[Cube, CubeList], template_cube: Union[Cube, CubeList]
-    ) -> Union[Tuple[Union[Cube, CubeList]], Cube, CubeList]:
+        self, *cubes: Union[Cube, CubeList]) -> Union[Cube, CubeList]:
```

Comment:
Ideally I would define this as:
`(template_cube, *cubes)`

However, this would mean swapping the order of edges in configurations and a cause of frustration for people migrating away from the CLI.